Improve README.md with latest changes (#534)
AikidoSec/firewall-node: README.md

Zen by Aikido is an embedded Web Application Firewall that autonomously protects Node.js apps against common and critical attacks.

> It protects your Node.js apps by preventing user input containing dangerous strings, which allow injection, pollution, and path traversal attacks. It runs on the same server as your Node.js app for simple [installation](#installation) and zero maintenance.

Reviewer: This sentence is still used in the other repos (e.g. python).

Author: Indeed, once this merged will change and streamline the README.md, because e.g. I don't think the LICENSE section is same everywhere

> It protects your Node.js apps by scanning incoming data and where that data flows to, allowing Zen to more accurately block SQL Injections, Path traversal attacks, and more. It runs on the same server as your Node.js app for simple [installation](#installation) and zero maintenance.

Author: ```suggestion
It protects your Node.js apps by scanning user input and where that data eventually flows to, allowing Zen to more accurately block SQL Injections, Path traversal attacks, and more. It runs on the same server as your Node.js app for simple [installation](#installation) and zero maintenance.
```

Zen operates autonomously on the same server as your Node.js app to:

* ✅ Secure your app like a classic web application firewall (WAF), but with none of the infrastructure or cost.
* ✅ Auto-generate API specifications

> * ✅ Block known threat actors and bots.

Author: ```suggestion
* ✅ Block known threat actors and bots.
* ✅ Geo-fencing to block or allow a selection of countries
```

> Aikido is your no nonsense application security platform. One central system that scans your source code & cloud, shows you what vulnerabilities matter, and how to fix them - fast. So you can get back to building.

> Zen improves Aikido's offering by providing you with security in production. Use the automatic generation of API specifications together with our DAST offering for even better API Security.

Reviewer 2: Replace DAST with API scanning or something similar?

Reviewer 2: I had no idea what DAST was and we want to bring these features to more engineers, so using industry terms will just confuse.

Reviewer: ```suggestion
Zen improves Aikido's offering by providing you with security in production. Use the automatic generation of API specifications together with our API scanning offering for even better API Security.
```

Only the necessary data gets reported back to Aikido, we scan your requests locally and only report back data once every 10 minutes.
If an attack on your application is detected, we report immediately allowing you to take swift action.

You can easily select which IP addresses and/or bots to block from curated lists inside our Dashboard.

> Running Zen locally without Aikido is possibly but the feature set is limited and we don't recommend this setup for production environments.

Author: ```suggestion
```
